src/applications/check-in/day-of/pages/ArrivedAtFacility.jsx

```diff
@@ -16,40 +19,91 @@ const ArrivedAtFacility = props => {
   const { goToNextPage, getCurrentPageFromRouter } = useFormRouting(router);
   const currentPage = getCurrentPageFromRouter();
 
+  const pageType = 'arrived-at-facility';
+
+  const testID = 'arrived-page';
+
   const bodyText = (
     <>
       <p>{t('we-ask-this-question-because-staff-can-call-you')}</p>
     </>
   );
 
-  const yesAction = () => {
+  // const bodyText = t('we-ask-this-question-because-staff-can-call-you');
+
+  const selectCurrentContext = useMemo(makeSelectCurrentContext, []);
+  const { setECheckinStartedCalled } = useSelector(selectCurrentContext);
+  const selectApp = useMemo(makeSelectApp, []);
+  const { app } = useSelector(selectApp);
+
+  const onClick = event => {
+    const answer = event.target.attributes.value.value;
     recordEvent({
-      event: createAnalyticsSlug('check-in-arrived-at-facility-yes'),
+      event: createAnalyticsSlug(
+        `${answer}-to-${pageType}${
+          setECheckinStartedCalled || app !== APP_NAMES.CHECK_IN ? '' : '-45MR'
+        }-clicked`,
+        'nav',
+        app,
+      ),
+    });
+  };
+
+  const yesAction = event => {
+    recordEvent({
+      event: createAnalyticsSlug('arrived-at-facility-yes'),
       fromPage: currentPage,
     });
-    dispatch(recordAnswer({ 'arrived-at-facility': 'yes' }));
+    dispatch(recordAnswer({ [pageType]: 'yes' }));
+    onClick(event);
     goToNextPage();
   };
 
-  const noAction = () => {
+  const noAction = event => {
     recordEvent({
-      event: createAnalyticsSlug('check-in-arrived-at-facility-no'),
+      event: createAnalyticsSlug('arrived-at-facility-no'),
       fromPage: currentPage,
     });
-    dispatch(recordAnswer({ 'arrived-at-facility': 'no' }));
+    dispatch(recordAnswer({ [pageType]: 'no' }));
+    onClick(event);
     goToNextPage();
   };
 
   return (
-    <ArrivedPage
-      header={t('have-you-arrived-at-your-va-health-facility')}
-      eyebrow={t('check-in')}
-      bodyText={bodyText}
-      yesAction={yesAction}
-      noAction={noAction}
-      pageType="arrived-at-facility"
-      router={router}
-    />
+    <>
+      <Wrapper
+        pageTitle={t('have-you-arrived-at-your-va-health-facility')}
+        classNames="travel-page"
+        testID={testID}
+      >
+        {bodyText && (
```

Comment:
Looks like bodyText always exists, so you could get rid of the var and move the jsx down here